### ISSUE TITLE
[PvE Mechanics] Steamvaults heroic buggs

### ISSUE THREAD

**Issue by descendance** [ccshiro/cc-buglist#179]
## Hydromancer Thespia-Those 2 water elementals which come with the boss do not respawn if you kill them and than wipe on the boss.

Slavemasters: these mobs do not command surounding slaves when pulled
How it should work: if they are near a group of slaves when you pull them you should also aggro the slaves who will fight you until you kill the slavemaster
source: https://web.archive.org/web/20070408104915/http://www.wowwiki.com/Steamvault

Mekgineer Steamriegger-when this boss casts his Electrified net he continues to channel it.
Not 100% sure but i think it should not be a channeling spell
Also he does not seem to channel his ability 'Shrink Ray' mby these two spell got mixed up?
source:
https://www.youtube.com/watch?v=OdWQYk6ZK-4&list=UUUjuRH2D-Krf9nC46fH7luA
https://web.archive.org/web/20070407071604/http://www.wowwiki.com/Mekgineer_Steamrigger

---

Myrmidon-these mobs cast Cleave on the tank but as it seems it does not do aoe damage, and only hits the tank.
Not sure that they have execute and sweeping strikes, might be wrong on this one.
source: https://web.archive.org/web/20070408104915/http://www.wowwiki.com/Steamvault

---

Oracle: Has a aoe silence for 5 seconds it should be 10 seconds
source: https://web.archive.org/web/20070408104915/http://www.wowwiki.com/Steamvault

Her heals also heal for like 5% of her hp.
How it should work: it should heal her fully or at least more than 5%
source: https://web.archive.org/web/20070408104915/http://www.wowwiki.com/Steamvault

---

Warlord Kalithresh-he buggs out when channeling those water tanks.
What happened:
-When he went to the first tank he started channeling his spell, after 1-2 seconds he casted 'Spell reflect shield' on himslef and broke the channeling spell
-On the second tank he went to, he again started channeling the spell but again after a few seconds he stopped. Tho he did not spell reflect himself this time, he just stopped channeling for no apparent reason.
How it should work: he should channel the tanks for a some time and if not destroyed fast enough he will enrage.
source: https://www.youtube.com/watch?v=3D-uEg4dwsU&list=UUUjuRH2D-Krf9nC46fH7luA

also: https://web.archive.org/web/20070407065148/http://www.wowwiki.com/Warlord_Kalithresh

**Comment by ccnim** [comment]
• Hydromancer Thespia
Worked fine for me on normal did not work on heroic however. Related to the issue we had with Nexus-Prince Shaffar?
• Slavemasters
Worked fine for me. Could you specify which mob?
• Mekgineer Steamrigger
His net should be channeled.
Shrink Ray has been tested and confirmed working.
• Coilfang Myrmidon 
Cleave tested and confirmed working.
Added Sweeping Strikes to his repertoire. 
• Oracle
AoE silence confirmed broken and fixed.
Heal issue is due to spell damage level calc.
• Warlord Kalithresh
Tested but could not reproduce.

**Comment by descendance** [comment]
About the slavemasters, it was those 2 solo patroling ones just before Mekgineer Steamrigger. There are groups of 2 up to 4 slaves and when we pulled the slavemaster he was next a group of 4 slaves but only 1 slave out of them aggroed us. If this helps in any way xD kinda hard to explain.

The warlord Kalithresh bug, i am not sure how i can help any more, we did not do anything special when fighting him.
